addon/components/abstract-input.js

```diff
@@ -56,9 +56,9 @@ export default Component.extend(PropTypeMixin, {
   // ==========================================================================
 
   @readOnly
-  @computed('errorMessage', 'showErrorMessage')
-  renderErrorMessage (errorMessage, showErrorMessage) {
-    if (!showErrorMessage) {
+  @computed('errorMessage', 'showErrorMessage', 'store.showAllErrors')
+  renderErrorMessage (errorMessage, showErrorMessage, showAllErrors) {
+    if (showAllErrors !== true && !showErrorMessage) {
```

Comment:
Is there a reason this can't just be 
```js
if (!showAllErrors && !showErrorMessage) {
```
Why do we need the explicit check against `true`?